.github/workflows/maven-deploy.yml

```diff
@@ -33,9 +34,9 @@ jobs:
       - name: Check Maven and Java versions
         run: mvn -v
 
-      - run: mvn clean deploy --batch-mode --activate-profiles release -Dgpg.signer=bc
+      - run: mvn -X -Dorg.slf4j.simpleLogger.log.org.apache.http.headers=debug -Dorg.slf4j.simpleLogger.log.org.apache.maven.wagon.providers.http.httpclient=debug clean deploy --batch-mode --activate-profiles release -Dgpg.signer=bc
         env:
-          NEXUS_USERNAME: ${{ secrets.nexus_username }}
-          NEXUS_PASSWORD: ${{ secrets.nexus_password }}
+          NEXUS_USERNAME: ${{ secrets.maven_username }}
+          NEXUS_PASSWORD: ${{ secrets.maven_password }}
```

Comment:
Are the repository secrets `NEXUS_PASSWORD` and `NEXUS_USERNAME` going to be obsolete? I'm not suggesting you change anything about these lines, but I just want to understand the status of the repo secrets.